enterprise/api/v1/serializers.py

```diff
@@ -1208,19 +1212,26 @@ def save(self):  # pylint: disable=arguments-differ
         self.validated_data['uuid'] = obj.uuid
 
 
-class EnterpriseCustomerInviteKeyReadOnlySerializer(serializers.ModelSerializer):
+class EnterpriseCustomerInviteKeyUpdateSerializer(BaseEnterpriseCustomerInviteKeySerializer):
+    """
+    Serializer for updating the EnterpriseCustomerInviteKey model.
+    """
+
+    uuid = serializers.UUIDField(read_only=True)
+    enterprise_customer_uuid = serializers.UUIDField(read_only=True)
+    usage_limit = serializers.IntegerField(read_only=True)
+    expiration_date = serializers.DateTimeField()
+    is_active = serializers.BooleanField()
+
+
+class EnterpriseCustomerInviteKeyReadOnlySerializer(BaseEnterpriseCustomerInviteKeySerializer):
     """
     Serializer for reading the EnterpriseCustomerInviteKey model.
     """
 
-    class Meta:
-        model = models.EnterpriseCustomerInviteKey
-        fields = (
-            'uuid',
-            'enterprise_customer_uuid',
-            'enterprise_customer_name',
-            'is_valid'
-        )
+    class Meta(BaseEnterpriseCustomerInviteKeySerializer.Meta):
+        fields = BaseEnterpriseCustomerInviteKeySerializer.Meta.fields + \
+            ('enterprise_customer_uuid', 'enterprise_customer_name')
```

Comment:
nit: `enterprise_customer_uuid` is already in the base class meta's fields.  Or does the `SerializerMethodField` not work without this here?

Reply:
it works without, good call